docs/src/examples/components/Popup/Types/PopupFocusTrapExample.shorthand.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { Button, Input, Header, Popup } from '@stardust-ui/react'
+
+const PopupFocusTrapExample = () => (
+  <>
+    <Popup
+      trigger={<Button icon="expand" content="Default trap behavior" />}
+      content={{ content: <Input icon="search" placeholder="Search..." /> }}
+    />
+
+    <Popup
+      trigger={<Button icon="expand" content="Lets trap your focus even more ツ" />}
+      focusTrap={{
```

Comment:
we kind of duplicating logic which is done in the Portal, for the exposing ```focusTrap``` prop.
So the original idea is
- Popup is not focus trapped by default
- FocusTrapZone props are coming from a behavior same way as for FocusZone.
Of course it's part of this PR, but what we need is to read FocusTrapZone props from ```accessibility.focusTrapZone = {} ```

After your PR merged, I will create PR that ads FocusTrapZone props to PopupBehavior.

Reply:
have discussion with @sophieH29, agreed on the following plan:
- prepare branch where focus trap properties will be read from behavior by `Popup`
- merge necessary changes to introduce this behavior into this PR
- remove `focusTrap` prop from the API of `Popup`